### DISCUSSION TITLE
onDoubleClick()

### DISCUSSION THREAD

**Discussion by amyspark-ng** [kaplayjs/kaplay#3] [Ideas]
I'd be nice to have an onDoubleClick event, taking as a parameter the time that could be between the two clicks, and also having a isMouseDoublePressed.

Also idk if this is being worked on, but events such as onClick should take a 'mouseButton' parameter, since i think right now it is only supported on functions that run onUpdate, such as isMousePressed

**Comment by lajbel** [comment]
yes

**Comment by niceEli** [comment]
Turned into feature request in #18
